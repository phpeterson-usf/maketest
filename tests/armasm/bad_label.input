-s tests/armasm/bad_label.s
