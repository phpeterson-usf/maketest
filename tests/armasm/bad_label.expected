Error: Unexpected newline after 'foo'. Expected a ':'
